Packs/OracleCloudInfrastructure/Integrations/OracleCloudInfrastructureEventCollector/OracleCloudInfrastructureEventCollector.yml

```diff
@@ -0,0 +1,81 @@
+category: Analytics & SIEM
+commonfields:
+  id: OracleCloudInfrastructureEventCollector
+  version: -1
+configuration:
+- display: Tenancy OCID
+  name: tenancy_ocid
+  required: true
+  type: 0
+  section: Connect
+- display: User OCID
+  name: user_ocid
+  required: true
+  type: 0
+  section: Connect
+- display: API Key Fingerprint
+  name: key_fingerprint
+  required: true
+  type: 0
+- display: Private Key
+  additionalinfo: May also be known as 'key file' or 'key content'.
+  name: private_key
+  required: true
+  type: 4
```

Comment:
Should the`private_key` type be Encrypted (4) or Authentication (9)?

Reply:
API Key Fingerprint and API Private Keywas changed to type 9 (authentication parameters).